test/test_maker.c

```diff
@@ -0,0 +1,69 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <sys/wait.h>
+
+#define LINE_LEN  1024
+#define TM_INCLUDE   "   test_maker::include\n"
+
+void raise_err(char *err_msg);
+
+int main(int argc, char **argv) {
+   if (argc != 2)
+      raise_err(
+         "test_maker: usage:\n"
+         "\t./test_maker\n"
+         "\t\tPATH_TEST_CASES");
+
+   FILE *src, *dest;
+
+   src = fopen("test.c", "r");
+   if (src == NULL)
+      raise_err("test_maker: failed to open test.c.");
+   dest = fopen("test.modified.c", "w+");
+   if (dest == NULL)
+      raise_err("test_maker: failed to open test.modified.c.");
+   
+   /* Write test.modified.c. */
+   char *line;
+
+   line = malloc(LINE_LEN);
```

Comment:
Why not `malloc(LINE_LEN * sizeof(char))`?
Anyway, I think the size of `line` should be variable. (realloc)